src/PubSubClient.cpp

```diff
@@ -166,9 +166,10 @@ bool PubSubClient::connect(const char* id, const char* user, const char* pass, c
                     v = v | (0x80 >> 1);
                 }
             }
+            uint16_t keepAlive = this->keepAliveMillis / 1000;
```

Comment:
Can be made `const uint16_t`

Reply:
Yes, but makes no real difference (same storage, compiler just checks that it's not altered any more). I'll do it.